### ISSUE TITLE
Item Set Tree display all sets

### ISSUE THREAD

**Issue by nathanxmeyer** [biblibre/omeka-s-module-ItemSetsTree#14]
I have a new Omeka 4 installation.  I have previously installed Item Set Tree module, built 4 items sets and arranged them in a hierarchy.  I then successfully built a web page displaying the set hierarchy as wanted using the setting "Display only sets selected on the Resources tab and their descendents."  Later I added three more sets that are unrelated to the first four (not in that hierarchy).  Now the previously built web page displays all seven sets.  I have de-activated and re-activated the module.  I have uninstalled using the EasyAdmin panel and separately by downloading the 0.7.0.zip file.  I have deleted and rebuilt the page.  I have switched display settings.  None of these steps prevents all 7 of the Item Sets from displaying on the page which is meant to only show 4.

**Comment by nathanxmeyer** [comment]
Just to confirm, I looked in the database and the data seems fine.

**Comment by nathanxmeyer** [comment]
Update on this.  When I used the prebuilt navigation page (the item set tree selected under Navigation) the tree displays correctly.  However, that display does not show item counts.  When I create a page from scratch and and the Item Set Tree block, it shows the additional (not selected sets) but does provide the counts.  Ideally I would like the counts.

**Comment by jajm** [comment]
It should be fixed now in `master` branch. Can you test and confirm ?

**Comment by nathanxmeyer** [comment]
Hello - I removed the old version from the modules directory, downloaded the Master branch as a zip, unzipped to the modules directory, and then renamed it.  In the Modules tab it showed as already installed and I clicked the configuration button to check it (was the same as previous module). I added a new page to my site and added the block for ItemSetsTree.  It displays only the sets selected for the site and has the correct item count.  The fix appears to work fine.  Thank you very much.
